docs/medical-api/handling-data/working-with-cohorts.mdx

```diff
@@ -0,0 +1,165 @@
+
+Patients can be added to cohorts on creation, or added to and removed from different cohorts as needed.
+
+During creation:
+
+- In [Bulk Patient Create](/medical-api/api-reference/patient/bulk-create-patient) there are `cohort1` and `cohort2` column headers for specifying the starting cohorts for a patient. See [here](/medical-api/api-reference/patient/bulk-create-patient#param-cohort1).
+- In [Create Patient](/medical-api/api-reference/patient/create-patient) there is a `cohorts` array parameter to set any number of cohorts on a patient. See [here](/medical-api/api-reference/patient/create-patient#param-cohorts).
+
+As needed:
+
+- Refer to endpoint [POST - Add Patients To Cohort](/medical-api/api-reference/cohort/add-patients-to-cohort)
+- Refer to endpoint [DELETE - Remove Patients From Cohort](/medical-api/api-reference/cohort/remove-patients-from-cohort)
+
+### Monitoring patients with Cohorts
+
+Metriport allows you to monitor patients and receive notifications about them throughout their care.
+
+To do this, you create a cohort and specify the settings directly on the cohort. Then add patients that need to be monitored on the cohort, and they will be opted-in to the monitoring settings of the cohort.
+
+```typescript
+// 1. Create a cohort
+const { cohort } = await metriportApi.createCohort({
+  name: "My Custom Cohort"
+  description: "This is my first cohort",
+  color: "purple",
+  settings: {
+    adtMonitoring: true
+  }
+ });
+
+// 2. Add patients to the cohort
+const patientsInCohort = await metriportApi.addPatientsToCohort({
+  cohortId: cohort.id,
+  patientIds: [
+    "patient-uuid-1",
+    "patient-uuid-2",
+    "patient-uuid-3",
+  ]
+});
+
+// 🎉 Your patients are now subscribed to ADT notifications!
+```
+
+<Tip>
+  Setting a patient's cohorts as you create them is the
+  easiest way + recommended way of enabling monitoring.
+</Tip>
+
+Alternatively, just create your patients as usual (via API call or bulk patient create) and specify the desired cohorts.
+
+```typescript
+// Create a patient and assign to existing cohorts
+const patient = await metriportApi.createPatient(
+  {
+    firstName: "John",
+    lastName: "Doe",
+    dob: "1990-01-01",
+    genderAtBirth: "M",
+    address: [
+      {
+        addressLine1: "123 Main St",
+        city: "Anytown",
+        state: "CA",
+        zip: "12345",
+        country: "USA",
+      },
+    ],
+    cohorts: ["High Risk", "My Custom Cohort"], // Assign cohorts during creation
+  },
+  facilityId
+);
+
+// The patient is now automatically subscribed to all monitoring settings
+// from both the "High Risk" and "My Custom Cohort" cohorts
+```
+
+### Available monitoring settings
+
+We're actively building out additional monitoring capabilities. Today, these are the valid fields and values for cohort settings.
+
+```json
+{
+  "adtMonitoring": true,
+  "questMonitoring": true,
+  "adtMonitoring_overrideHealthConnectTexas": true,
+  "adtMonitoring_overrideTexasHiePcc": true
+}
```

Comment:
let's work on the data model here, IMO this is verbose and not very dev friendly - what do we think of something like:

```
{
    "monitoring":
        {
            "adt": true,
            "pharmacy": true,
            "lab": true,
            "hie": true
        },
   "overrides":
   {
        hie: {
            "healthConnectTexas": true
        }
   }
}
```